src/user/image.ts

```diff
@@ -0,0 +1,64 @@
+import { instanceToClass } from "clone-class"
```

Comment:
Please run `npm run lint:es` before you send Pull Request, because we need to follow the original code styles.